packages/@sfx/search-plugin/test/unit/common/search-plugin.test.ts

```diff
@@ -0,0 +1,63 @@
+import { expect, sinon, stub } from '../../utils';
+import SearchPlugin from '../../../src/search-plugin';
+import { SearchPluginOptions, SearchPluginExposedValue } from '../../../src/search-plugin';
+import { BridgeConfig, Query } from 'groupby-api';
+import * as SearchPackage from 'groupby-api';
+
+describe('SearchPlugin', () => {
+  let searchPlugin: any;
+
+  beforeEach(() => {
+    searchPlugin = new SearchPlugin({ customerId: 'testClientId', https: true });
+  });
+
+  describe('metadata getter', () => {
+    it('should have the name `search`', () => {
+      expect(searchPlugin.metadata.name).to.equal('search');
+    });
+
+    it('should not specify any dependencies', () => {
+      expect(searchPlugin.metadata.depends).to.deep.equal([]);
+    });
+  });
+
+  describe('constructor()', () => {
+    it('should default https to true', () => {
```

Comment:
This is a good test to have, but I would approach it differently. In general, you don't want to introduce and expose structures solely for testing purposes.

I would take an approach similar to your third test: stub `BrowserBridge` and assert that it was called with an `https` value of `true`.